javascript/index.js

```diff
@@ -14,52 +16,91 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  printSeconds();
+  printMinutes();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minUni.innerHTML = (chronometer.getMinutes())%10;
```

Comment:
You could only run the function once to get the currentTime in a variable and then apply the %10 or /10 for each dom element. It is an additional line but only calling the function once would be better for performance.